RoboJackets/robocup-software
# Add Automated Testing Framework

## Description
https://docs.google.com/document/d/1jem3jhoX-xXPAJe7aBImg3YXuQ82eKvUvw6GmLgNdgg/edit?usp=drive_web&ouid=107140160100665177019

## Associated Issue
Issue #1328 

## Design Documents
[Link](https://docs.google.com/document/d/1jem3jhoX-xXPAJe7aBImg3YXuQ82eKvUvw6GmLgNdgg/edit?usp=drive_web&ouid=107140160100665177019)

## Steps to test
### Test Case 1
1. Select the Testing Tab
2. Click the checkmark next to the example test
3. Click add
4. Click Run or Next

Expected result: 2 blue robots and 2 yellow robots will appear on the opposite side of the field as the rest of the team and the ball will move to a new location with a significant velocity assigned. Additionally, Basic 122 will begin running and test pivot kick will also be selected.
